doc/examples/cloud-config-chef.txt

```diff
@@ -52,6 +52,9 @@ apt:
 
 chef:
 
+  # Valid values are 'accept' and 'accept-no-persist'
```

Comment:
Reading https://docs.chef.io/chef_license_accept/, I don't think that `accept-no-persist` is a valid value for this configuration; it looks like it affects whether or not the CLI tooling will write `chef_license: "accept"` when run.

Reply:
Reading https://docs.chef.io/config_rb_client/ documents additional configuration options,
as documented

chef_license
Can be set to "accept" or "accept-no-persist"

Reply:
You're absolutely right, I misread; apologies!